src/components/Header/Header.stories.mdx

```diff
@@ -0,0 +1,84 @@
+import { Story, Canvas, Props } from '@storybook/addon-docs/blocks';
+import Header from './Header.jsx';
+
+<Meta title="Header" component={Header} />
+
+# Header
+
+Very large text that can be used to present the title and short additional intro text for a non-article page.
+
+It takes optional children that should be used for relevant non-text components, such as navigation elements.
+
+## Usage 
+
+```js
+import { Header } from '@quartz/interface/src/components';
+```
+
```

Comment:
fyi i've been removing this, very redundant. i think i'll add it to the welcome story
```suggestion
```